api/src/pcapi/alembic/versions/20250107T160202_bfd459b4a913_special_event_event_date_not_null.py

```diff
@@ -0,0 +1,37 @@
+"""set special_event.eventDate to NOT NULL"""
```

Comment:
Détail : mettre à jour le texte pour correspondre aux nouveaux changements 
```suggestion
"""Recreate column special_event.eventDate as DATE NOT NULL"""
```